app/lib/Plugins/Media/Office.php

```diff
@@ -308,7 +308,7 @@ private function isWordExcelorPPTdoc($ps_filepath) {
 		}
 		
 		// 2007+ .docx files
-		if (in_array(pathinfo(strtolower($ps_filepath), PATHINFO_EXTENSION), ['doc', 'docx'])) {	// PhpWord often will identify Excel docs as Word (and PHPExcel will identify Word docs as Excel...) so we test file extensions here			
+		if (in_array(pathinfo(strtolower($ps_filepath), PATHINFO_EXTENSION), ['doc', 'docx'])) {	// PhpWord often will identify Excel docs as Word (and \PhpOffice\PhpSpreadsheet\Spreadsheet will identify Word docs as Excel...) so we test file extensions here			
```

Comment:
I suggest change the inline comment to a line comment before the code it refers to.